package.json

```diff
@@ -18,6 +18,10 @@
     "webrtc-adapter": "8.2.2"
   },
   "devDependencies": {
-    "eslint": "8.35.0"
+    "eslint": "8.35.0",
+    "eslint-plugin-html": "^7.1.0"
```

Comment:
Could you please:
- upgrade eslint to the latest version `8.39.0`
- use a fixed version of the `eslint-plugin-html`

Reply:
Fixing the package versions is typically done by committing the package-lock.json file, which also specifies the versions of any indirect dependencies. If you want fixed package versions I recommend doing that

Reply:
I agree a package lock is a more convenient way of fixing packets, but that can be done in a separate effort.
Since eslint is already fixed to a specific version, I'd just want to align the linter and the plugin versions.